xml/common_intro_typografie_i.xml

```diff
@@ -11,20 +11,59 @@
  <title>Documentation Conventions</title>
 
  <info>
-<dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
-    <dm:maintainer>fs</dm:maintainer>
-    <dm:status>editing</dm:status>
-    <dm:deadline/>
-    <dm:priority/>
-    <dm:translation>no</dm:translation>
-    <dm:languages/>
-</dm:docmanager>
-</info>
-<para>
-  The following typographical conventions are used in this manual:
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:maintainer>fs</dm:maintainer>
+   <dm:status>editing</dm:status>
+   <dm:deadline/>
+   <dm:priority/>
+   <dm:translation>no</dm:translation>
+   <dm:languages/>
+  </dm:docmanager>
+ </info>
+
+ <para>
+  The following admonition elements and typographical conventions are used
+  in this documentation:
  </para>
 
  <itemizedlist mark="bullet" spacing="normal">
+  <warning>
+   <para>
+    Vital information to be aware of before proceeding. Warns you about
+    security issues, potential loss of data, damage to hardware, or physical
+    hazards.
+   </para>
+  </warning>
+  <important>
+   <para>
+    Important information to be aware of before proceeding.
+   </para>
+  </important>
+  <note>
+   <para>
+    Additional information, for example about differences in software
+    versions.
+   </para>
+  </note>
+  <tip>
+   <para>
+    A guideline or a piece of practical advice.
```

Comment:
What do you think about something like "Optional information"? IMHO your could emphasis the optional nature of a tip more (it can be ignored, it's not essential).